forms/pdf_form_fill_json.go

```diff
@@ -133,23 +133,18 @@ func fillFields(inputPath, jsonPath, outputPath string) error {
 		return err
 	}
 
-	// Write out.
-	pdfWriter := model.NewPdfWriter()
-	pdfWriter.SetForms(nil)
-
-	for _, p := range pdfReader.PageList {
-		err := pdfWriter.AddPage(p)
-		if err != nil {
-			return err
-		}
+	// AcroForm field no longer needed on the new PDF.
+	opt := &model.ReaderToWriterOpts{
+		SkipAcroForm: true,
```

Comment:
Quick note: `SkipAcroForm` should rarely be set to true (only in cases where a user wants to throw away or replace the AcroForm). It should not be set to true when filling or flattening forms because if there's nothing left in the AcroForm after the flattening process, the AcroForm of the reader will be `nil`. However, if not all fields are flattened, the AcroForm will not be `nil` and it should not be skipped.

Reply:
Same as previous reply, the example code I'm replacing is setting the AcroForm to nil. So I'm skipping AcroForm here to achieve the same result.